examples/planning/open_loop_test.ipynb

```diff
@@ -0,0 +1,345 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# ===== INIT DATASET\n",
+    "eval_cfg = cfg[\"val_data_loader\"]\n",
+    "rasterizer = build_rasterizer(cfg, dm)\n",
+    "eval_zarr = ChunkedDataset(dm.require(eval_cfg[\"key\"])).open()\n",
+    "eval_dataset = EgoDataset(cfg, eval_zarr, rasterizer)\n",
+    "eval_dataloader = DataLoader(eval_dataset, shuffle=eval_cfg[\"shuffle\"], batch_size=eval_cfg[\"batch_size\"], \n",
+    "                             num_workers=eval_cfg[\"num_workers\"])\n",
+    "print(eval_dataset)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Evaluation loop\n",
+    "\n",
+    "Here, we loop through the data and store predicted and annotated trajectories (positions + yaws).\n",
+    "\n",
+    "Note: we're not taking into account availability here. We acknowledge this can reflects in lower metric."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# ==== EVAL LOOP\n",
+    "position_preds = []\n",
+    "yaw_preds = []\n",
+    "\n",
+    "position_gts = []\n",
+    "yaw_gts = []\n",
+    "\n",
+    "torch.set_grad_enabled(False)\n",
+    "idx = 0\n",
+    "\n",
+    "for data in tqdm(eval_dataloader):\n",
+    "    del data[\"host_id\"]\n",
+    "    result = model(data)\n",
+    "    position_preds.append(result[\"positions\"].detach().cpu().numpy())\n",
+    "    yaw_preds.append(result[\"yaws\"].detach().cpu().numpy())\n",
+    "\n",
+    "    position_gts.append(data[\"target_positions\"].detach().cpu().numpy())\n",
+    "    yaw_gts.append(data[\"target_yaws\"].detach().cpu().numpy())\n",
+    "    idx += 1\n",
+    "    if idx == 10:  # TODO remove\n",
+    "        break\n",
+    "    \n",
+    "position_preds = np.concatenate(position_preds)\n",
+    "yaw_preds = np.concatenate(yaw_preds)\n",
+    "\n",
+    "position_gts = np.concatenate(position_gts)\n",
+    "yaw_gts = np.concatenate(yaw_gts)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Quantitative evaluation: ADE, FDE and angle distance\n",
+    "\n",
+    "### Positional displacement\n",
+    "[Average Displacement Error (ADE) and Final Displacement Error (FDE)](https://en.wikipedia.org/wiki/Mean_squared_displacement) are standard metrics used to evaluate future predictions for AVs.\n",
+    "\n",
+    "We can compute them by comparing predicted and annotated positions, which we have stored in the previous cell.\n",
+    "Additionally, we can plot histograms of their distributions across samples to better capture the variance of our error\n",
+    "\n",
+    "### Angle displacement\n",
+    "\n",
+    "For the yaw, we can use the Minimum Angle Distance to check the error. Again, we can plot a histogram to inspect the error distribution."
```

Comment:
I'm still missing some motivation here - why do I need to get angle errors (-> spot if car is turning wrongly e.g.?)

Reply:
Added a sentence, tell me if it's better now